server/routes/referrals/complexityLevelPresenter.test.ts

```diff
@@ -91,9 +69,9 @@ describe('ComplexityLevelPresenter', () => {
 
   describe('title', () => {
     it('returns a title', () => {
-      const presenter = new ComplexityLevelPresenter(draftReferral, socialInclusionComplexityLevels)
+      const presenter = new ComplexityLevelPresenter(draftReferral, serviceCategory)
 
-      expect(presenter.title).toEqual('What is the complexity level for the social inclusion service?')
+      expect(presenter.title).toEqual('What is the complexity level for the accommodation service?')
```

Comment:
Same as other comment - I think better not to rely on the values in the factory, where possible, and to explicitly set the name of the service category when creating it.